packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -1,22 +1,33 @@
 /**
- * When user wants to bridge ETH from L1 to L2
+ * When user wants to bridge ERC20 from L1 to L2
  */
 
+import { utils } from 'ethers'
 import { formatAmount } from '../../../src/util/NumberUtils'
 import { resetSeenTimeStampCache } from '../../support/commands'
 import {
   ERC20TokenAddressL1,
   getInitialERC20Balance,
-  goerliRPC,
+  ethRpcUrl,
+  l1NetworkConfig,
   zeroToLessThanOneETH
 } from '../../support/common'
 
 describe('Deposit ERC20 Token', () => {
   // when all of our tests need to run in a logged-in state
   // we have to make sure we preserve a healthy LocalStorage state
   // because it is cleared between each `it` cypress test
+  const ERC20AmountToSend = 0.0001
 
   before(() => {
+    cy.log('Approving WETH...')
+    cy.approveWeth()
+    // we don't have any erc20 locally so we are wrapping some eth
+    cy.wrapEth(utils.parseEther(String(ERC20AmountToSend)))
+    cy.log('Wrapping some ETH...')
+    // makes sure weth reflects in the account
+    // eslint-disable-next-line
+    cy.wait(15000)
```

Comment:
This can be done in a nicer way, such as `cy.waitUntil(amount_visible_in_UI)` but that can be refactored later

Reply:
Can we have a github issue or notion ticket to track it? I feel like TODO without actionnable ticket are neven taken care of.

Reply:
I will just add a notion ticket and assign to myself

Reply:
@bartlomiej94 this should be done when tests set up, before Cypress starts. We have to tackle as part of this PR.